README.md

```diff
@@ -313,6 +313,7 @@ Parameter | Description
 Example:
 ```yaml
 ---
+safetyOperationEnabled: false
```

Comment:
can we change this to something simpler like 'safetyOn' or 'safeMode' to prevent confusion in naming. Otherwise lgtm